test/unit/view-layout/test-multiview.cpp

```diff
@@ -43,6 +43,13 @@ TYPED_TEST(MultiViewUnitTest, Constructors)
   RAJA::MultiView<TypeParam, layout> view(data, layout(10));
   ASSERT_EQ( val, view(0,0) );
 
+  /*
+   * Should be able to construct an empty MultiView
+   */
+  RAJA::MultiView<TypeParam, layout> view_empty;
+  ASSERT_EQ( nullptr, view_empty.get_data() );
+  ASSERT_EQ( 1, (view_empty.get_layout()).size() );
```

Comment:
How about adding a test for setting the layout after this?

Reply:
Done.